src/models/MySqlModel.ts

```diff
@@ -199,9 +222,10 @@ export class MySqlModel {
     table: string,
     id: number,
     reference = 'undefined caller',
-  ): Promise<number> {
+  ): Promise<boolean> {
```

Comment:
From what I observed, mySQL was not returning the data that was deleted. It was returning an object that looked like 
```
{
      fieldCount: 0,
      affectedRows: 1,
      insertId: 0,
      info: '',
      serverStatus: 2,
      warningStatus: 0,
      changedRows: 0
    }
```